libvcx/src/connection.rs

```diff
@@ -369,7 +371,7 @@ pub fn set_pw_did(handle: u32, did: &str) -> VcxResult<()> {
     CONNECTION_MAP.get_mut(handle, |connection| {
         match connection {
             Connections::V1(ref mut connection) => Ok(connection.set_pw_did(did)),
-            Connections::V3(_) => Err(VcxError::from(VcxErrorKind::InvalidConnectionHandle))
+            Connections::V3(ref mut connection) => Ok(connection.set_pw_did(did))
```

Comment:
I'd suggest to keep 
```
            Connections::V3(_) => Err(VcxError::from(VcxErrorKind::InvalidConnectionHandle))	
```
to preserve information about the fact that `set_pw_did` is not used in V3. We'll know we can safely delete any code using it, as it must imply it's V1 code.